std_msgs/Header header

uint8 characteristicSpeedErrAmp
uint8 qualifierCharacteristicSpeed
uint8 characteristicSpeed